components/authz-service/server/v2/project_update_manager_internal_test.go

```diff
@@ -1,51 +0,0 @@
-package v2
-
-import (
-	"testing"
-	"time"
-
-	"github.com/stretchr/testify/assert"
-)
-
-func TestFindLongestEstimatedTimeEmpty(t *testing.T) {
```

Comment:
This function was removed for the already existing `FindSlowestJobStatus` function.